src/app/Swasti/SponsorCompany.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import image1 from "./Assets/1.svg"
+import image2 from "./Assets/2.svg"
+import image3 from "./Assets/3.svg"
+
+import "./SponsorCompany.css"
+import Image from 'next/image'
+const SponsorCompany = () => {
+  const images = [image1, image2, image3]
+  return (
+    <>
+    <section className='main-section'>
+    
+    {
+      images.map((image, index) => {
```

Comment:
goodwork here ! but lets just take all the sponsor companies images to different jsx or json file and import and map from there and instead of using 3 images use all 9 images for now even if you have to repeat the images as it's gonna create problem later